automation/check-patch.yumrepos

```diff
@@ -0,0 +1,11 @@
+[fedora]
```

Comment:
There are a few things missing in this file for the the mirros to work:

1. There should probably be two repos here - 'base' and 'updates'.
2. The names of the repos that go in the square brackets are important - this is how the mirrors system identifies which mirrors to use. For fc29 the repo names should be `fedora-base-fc29` and `fedora-updates-fc29` respectively.
3. See comment below about the idea of including a `[main]` section in this file as well.